bookdown/03-pipelines.Rmd

```diff
@@ -40,6 +40,24 @@ A predecessor to this package is the [mlrCPO](https://github.com/mlr-org/mlrCPO)
 Other packages that provide, to varying degree, some preprocessing functionality or machine learning domain specific language, are the
 `r cran_pkg("caret")` package and the related `r cran_pkg("recipies")`  project, and the `r cran_pkg("dplyr")` package.
 
+An example for a Pipeline that can be constructed using
+`r mlr_pkg("mlr3pipelines")` is depicted below:
+
+```{r 03-pipelines-000, echo = FALSE, width = 10, height = 10}
+library(visNetwork)
+graph = mlr_pipeops$get("branch", c("null", "pca", "scale")) %>>%
+  gunion(list(
+      mlr_pipeops$get("null", id = "null1"),
+      mlr_pipeops$get("pca"),
+      mlr_pipeops$get("scale")
+  ))
+gr = graph %>>%
+  mlr_pipeops$get("unbranch", c("null", "pca", "scale")) %>>%
+  mlr_pipeops$get("learner", mlr_learners$get("classif.rpart"))
+
+visInteraction(zoomView = FALSE, gr$plot(html = TRUE))
+```
+
```

Comment:
I think the first code example of a pipeline should not directly start with a branching example. Why did you include this example here if the PR is about the regression use case?

Reply:
This is just a teaser figure, and we do not show any code here. I do not think this will confuse anybody.

Reply:
Ah I missed the "echo = FALSE".